truffle/src/main/java/org/jruby/truffle/nodes/rubinius/RandomPrimitiveNodes.java

```diff
@@ -0,0 +1,113 @@
+package org.jruby.truffle.nodes.rubinius;
+
+import com.oracle.truffle.api.CompilerDirectives;
+import com.oracle.truffle.api.dsl.Specialization;
+import com.oracle.truffle.api.source.SourceSection;
+import com.oracle.truffle.api.utilities.ConditionProfile;
+import org.jruby.Ruby;
+import org.jruby.truffle.runtime.RubyContext;
+import org.jruby.truffle.runtime.core.RubyBasicObject;
+import org.jruby.truffle.runtime.core.RubyBignum;
+
+import java.math.BigInteger;
+
+/**
+ * Rubinius primitives associated with the Ruby {@code Random} class.
+ */
+
+public abstract class RandomPrimitiveNodes {
+
+    @RubiniusPrimitive(name = "randomizer_seed")
+    public static abstract class RandomizerSeedPrimitiveNodes extends RubiniusPrimitiveNode {
+
+        private final ConditionProfile nullByteProfile = ConditionProfile.createBinaryProfile();
```

Comment:
You've got this `nullByteProfile` in all your nodes - is this left over from some copy and paste? You never use it.